src/js/vre/chapter36/config/form.js

```diff
@@ -6,19 +6,24 @@ import {
   genderLabels
 } from '../../../common/utils/labels.jsx';
 
+import * as address from '../../../common/schemaform/definitions/address';
```

Comment:
Make sure this is the correct schema to be used.

Reply:
It is, as far as I know.